packages/domain/tests/todos/todo.tests.js

```diff
@@ -177,6 +163,199 @@ describe("Todos.Todo", function() {
           })
         ]);
     });
+  });
+
+
+  describe("removing many todos", function() {
```

Comment:
We where testing the infrastructure (bug in snapshot creation). I agree that it is no longer needed since the assumption must be that infrastructure works.